ocaml/xapi-guard/test/cache_test.ml

```diff
@@ -0,0 +1,139 @@
+let ( let@ ) f x = f x
+
+let ( let* ) = Lwt.bind
+
+module Tpm = Xapi_guard.Types.Tpm
+
+module TPMs = struct
+  let request_persist uuid write =
+    let __FUN = __FUNCTION__ in
+
+    let key = Tpm.deserialize_key (Random.int 3) in
+
+    let time = Mtime_clock.now () in
+    let contents =
+      Printf.sprintf "contents %s" (Mtime.to_uint64_ns time |> Int64.to_string)
+    in
+    let* () =
+      Logs_lwt.app (fun m ->
+          m "%s: New request! %a/%i/%a" __FUN Uuidm.pp uuid
+            Tpm.(serialize_key key)
+            Mtime.pp time
+      )
+    in
+    write (uuid, time, key) contents
+end
+
+let lwt_reporter () =
```

Comment:
Don't we already have an equivalent of this in the xapi logs module?

Reply:
No, it's only on `wsproxy.ml`.

I've wanted to use Logs for logging years. Maybe we can get it cleaned up now that we have a monorepo. We might even get to the point of having no cost for unlogged loglines